index.html

```diff
@@ -0,0 +1,76 @@
+<html lang="en">
+<head>
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Tribute page</title>
+</head>
+<body>
+<main id ="main">
+<h1 id = "title" >President Barack Obama</h1>
+</div>
+<div id="img-div">
+<center> <img style = "display: block; max-width:100%" id = "image" src="https://www.whitehouse.gov/wp-content/uploads/2021/01/44_barack_obama.jpg?resize=640,640" alt="American president in white house" ></center>
+<div id = "img-caption">
+<h2>Barack Obama served as the 44th President of the United States. His story is the American story — values from the
+     heartland, a middle-class upbringing in a strong family, hard work and education as the means of getting ahead, 
+     and the conviction that a life so blessed should be lived in service to others.</h2>
+</div>
+
+</div>
```

Comment:
please try to minimize the usegage of <div> and instead embrace usage of semantic tags because it enhance 100%  more readable, effective for SEO and accessibility purposes.